autoload/codefmt/clangformat.vim

```diff
@@ -91,9 +108,9 @@ function! codefmt#clangformat#GetFormatter() abort
       return
     endif
 
-    let l:cmd = [
-        \ s:plugin.Flag('clang_format_executable'),
-        \ '-style', l:style]
+    " Split the command on spaces, except when there's a proceeding \
```

Comment:
I know this is copy/pasted, but I think you want "preceding" here.

Reply:
Fixed